flink-kubernetes/src/main/resources/META-INF/NOTICE

```diff
@@ -15,6 +15,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.squareup.okhttp3:okhttp:3.12.1
 - com.squareup.okio:okio:1.15.0
 - io.fabric8:kubernetes-client:4.5.2
+- io.fabric8:zjsonpatch:0.3.0
```

Comment:
Nit: Maybe we could put the dependencies in alphabetical order.

Reply:
Fixed by [64137c6](https://github.com/apache/flink/pull/11233/commits/64137c6034da191208680bbe89948d911eec6e52)